js/script_find_the_number.js

```diff
@@ -0,0 +1,57 @@
+let guess = 0;
+let randomNumber;
+let userNumber;
+let highestNumber;
+let setMaxTries;
+let userChoice;
+
+// Set how big should the random number be
+function setMaxNumber(){
+    highestNumber = document.getElementById("setMaxNumber").value;
+    setRandomNumber();
+    if (highestNumber <= 0){
+        createPopup("Please enter number bigger than 0", true);
+    }
+}
+
+// Set how many tries user wants to find the number
+function setMaxAttempts(){
+    setMaxTries = document.getElementById("setMaxAttempts").value;
+    if (setMaxTries <= 0){
+        createPopup("Please enter number bigger than 0",true);
+    }
+}
```

Comment:
Du rufst die 2 Funktionen nie auf - deswegen bekommst du auch keine Fehlermeldung wenn du 0 eingibst.
Rufe die 2 Funktion in startGame auf und verlange einen returnwert.
Zum Beispiel:
function startGame(){
if(setMaxNumber() && setMaxAttempts()) {
let activeDiv = document.getElementsByClassName("active")[0];
    activeDiv.classList.remove("active");
    activeDiv.nextElementSibling.classList.add("active");
}
}